pagopa/io-app-design-system
# [IOAPPX-289] Delete unused colors from `IOColors`

> [!note]
> This PR is closely related to the following PR:
> * https://github.com/pagopa/io-app/pull/5743

## Short description
This PR removes a number of unused color values from the main `IOColors` object and its related objects.

## List of changes proposed in this pull request
- Remove the following colors:
  - `cobalt`
  - `milderGray`
  - `noCieButton`
  - `yellow`

## How to test
1. Run the example app
2. Check the updated **Colors** page
